docs/source/quickstart/script.md

```diff
@@ -0,0 +1,262 @@
+|                               |                 |                 | ml4ir/config/keys.py                               |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --feature_config              | <class 'str'>   | None            | Path to YAML file or YAML string with feature      |
+|                               |                 |                 | metadata for training.                             |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --model_file                  | <class 'str'>   |                 | Path to a pretrained model to load for either      |
+|                               |                 |                 | resuming training or for running ininference mode. |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --model_config                | <class 'str'>   | ml4ir/base/conf | Path to the Model config YAML used to build the    |
+|                               |                 | ig/default_mode | model architecture.                                |
+|                               |                 | l_config.yaml   |                                                    |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --optimizer_key               | <class 'str'>   | adam            | Optimizer to use. Has to be one of the optimizers  |
+|                               |                 |                 | in OptimizerKey under ml4ir/config/keys.py         |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --loss_key                    | <class 'str'>   | None            | Loss to optimize. Has to be one of the losses in   |
+|                               |                 |                 | LossKey under ml4ir/config/keys.py                 |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --metrics_keys                | <class 'str'>   | None            | Metric to compute. Can be a list. Has to be one of |
+|                               |                 |                 | the metrics in MetricKey under                     |
+|                               |                 |                 | ml4ir/config/keys.py                               |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --monitor_metric              | <class 'str'>   | None            | Metric name to use for monitoring training loop in |
+|                               |                 |                 | callbacksml4ir/config/keys.py                      |
```

Comment:
copy / paste problem ? 
I guess `ml4ir/config/keys.py ` should not be there

Reply:
Hmm. Let me fix it.